requirements.txt

```diff
@@ -1,2 +1,2 @@
 Flask
-pytest
+pytestpip
```

Comment:
Nie wiem czemu mam to zmienione :( 
Jak usunąć ten plik, żeby nie wszedł potem do mastera?

Reply:
teraz jest już okey

Reply:
zostawiamy pytest